pkg/mcp/configz/server/configz_test.go

```diff
@@ -53,7 +53,11 @@ func TestConfigZ(t *testing.T) {
 
 	o := ctrlz.DefaultOptions()
 	cz, _ := ctrlz.Run(o, []fw.Topic{CreateTopic(s.Cache)})
-	defer cz.Close()
+	defer func() {
```

Comment:
should you do the if around the defer instead?

Reply:
Yeah I think I misunderstood what was happening here. Will put up a better fix in a second that dynamically assigns the port, since this is the root cause of the error (port conflict)